<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminsearch</class>
 <widget class="QDialog" name="adminsearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>管理员查询</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>起始站</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="start_station">
       <item>
        <property name="text">
         <string>北京</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>上海</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>西藏</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>终点站</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="end_station">
       <item>
        <property name="text">
         <string>北京</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>出发时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="time_station"/>
     </item>
     <item>
      <widget class="QPushButton" name="search_by_station">
       <property name="text">
        <string>按站查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>起始站</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="start_city">
       <item>
        <property name="text">
         <string>北京</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>上海</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>西藏</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>终点站</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="end_city">
       <item>
        <property name="text">
         <string>北京</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>出发时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="time_city"/>
     </item>
     <item>
      <widget class="QPushButton" name="search_by_city">
       <property name="text">
        <string>按城市查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="back_index">
         <property name="text">
          <string>回到主页</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item>
      <widget class="QScrollBar" name="verticalScrollBar">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
